tests/data/placebo/test_s3_bucket_encryption_filter_kms/kms.ListAliases_1.json

```diff
@@ -0,0 +1,25 @@
+{
+  "status_code": 200,
```

Comment:
please ensure recorded data only  contains dummy account ids, or the test account id used commonly in custodian, if your following the developer docs for recording, it should be automatically replacing any ids.

Reply:
I used what was already in placebo directory